assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/che/che.properties

```diff
@@ -543,8 +543,17 @@ che.singleport.wildcard_domain.ipless=false
 
 # Docker image of Che plugin broker app that resolves workspace tooling configuration and copies
 # plugins dependencies to a workspace
-che.workspace.plugin_broker.metadata.image=quay.io/eclipse/che-plugin-metadata-broker:v3.3.0
-che.workspace.plugin_broker.artifacts.image=quay.io/eclipse/che-plugin-artifacts-broker:v3.3.0
+#
+# Note these images are overridden by the Che Operator by default; changing the images here will not
+# have an effect if Che is installed via Operator.
+che.workspace.plugin_broker.metadata.image=quay.io/eclipse/che-plugin-metadata-broker:v3.4.0
+che.workspace.plugin_broker.artifacts.image=quay.io/eclipse/che-plugin-artifacts-broker:v3.4.0
+
+# Configures the default behavior of the plugin brokers when provisioning plugins into a workspace.
+# If set to true, the plugin brokers will attempt to merge plugins when possible (i.e. they run in
+# the same sidecar image and do not have conflicting settings). This value is the default setting
+# used when the devfile does not specify otherwise, via the "mergePlugins" attribute.
+che.workspace.plugin_broker.default_merge_plugins=false
```

Comment:
Is this set to `false` by default because of minimizing the possible changes in the behavior? If I understood the changes in the plugin broker, it should be fairly conservative about when it actually does merge the plugins or not.

Reply:
I set it to `false` because the intention was to merge this for the coming release -- I didn't want to risk introducing a regression with no time to fix it. My basic plan was

- `7.19.0`: merging available, can be enabled by setting attribute in devfile
- `7.20.0`: merging available and default, can be disabled by setting attribute in devfile

I generally try to get changes that can impact the default flow + workspace startup in earlier in a sprint so we can catch any unforeseen issues before releasing, so this is kind of a workaround.